scripts/run_network.py

```
@@ -0,0 +1,332 @@
+                
+                if not resolved_url:
+                    print(f"Error: Node '{node_name}' imports '{imp}', but '{imp}' has not been started yet.")
+                    exit_orchestrator(1, processes, f"Import resolution error for node '{node_name}'")
+
+                import_flags.extend(["-i", resolved_url])
+
+        if port.lower() == "client":
+            log_file_path = os.path.join(clients_log_dir, f"{node_name}.log")
+            # Client Node (runs in background)
+            print(f"[{node_name}] Starting client node running '{file_path}'...")
+            cmd = ["cargo", "run", "-p", "meerkat", "--", "--local", "-f", file_path] + import_flags
```

Ruff (0.15.15)

[warning] Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)